Lib/tkinter/font.py

```diff
@@ -100,6 +100,9 @@ def __init__(self, root=None, font=None, name=None, exists=False,
     def __str__(self):
         return self.name
 
+    def __repr__(self):
+        return f"{self.__class__.__module__}.{self.__class__.__name__} object '{self.name}'"
```

Comment:
Please add `<` and `>` in the string.

It is needed also to add a test and a NEWS entry (read in the Devguide how to use `blurb` for this). You can also add your name in `Misc/ACKS`.